### DISCUSSION TITLE
Is it possible to transcribe and then translate to language x?

### DISCUSSION THREAD

**Discussion by kodxana** [openai/whisper#22] [Q&A] [answered]
Currently translation system support translation only to english language.
Would it be possible to allow for example transcription from for example English audio to Polish text?

**Comment by knaik** [comment, marked as answer]
From the paper, the dataset that was used did not use any English audio to polish text samples. The dataset was cleaned by using a different model to match spoken language with text language. If they did not match, the sample was excluded. An exception was made for a portion of the training data to match any spoken language to English text (X->en) translation. 

So unfortunately there is no direct way, the model wasn't trained on it. For your use case, this can transcribe to English text, but there has to be some an outside system to translate from English text to Polish text.
